client/packages/programs/src/JsonForms/useJsonForms.tsx

```diff
@@ -209,5 +223,7 @@ export const useJsonForms = <R,>(
     isDirty: isDirty ?? false,
     error,
     validationError,
+    submitActions,
+    updateSubmitActions,
```

Comment:
Not actually doing anything with these return values currently, but could be handy for interacting with from outside. Not sure if will be useful, will remove if not necessary after #5264